packages/core/src/external/aws/lambda-logic/bundle-to-brief.ts

```diff
@@ -0,0 +1,108 @@
+import { Bundle } from "@medplum/fhirtypes";
+import { errorToString } from "@metriport/shared";
+import { capture, out } from "../../../util";
+import { findDiagnosticReportResources, findPatientResource } from "../../fhir/shared";
+import { BedrockUtils } from "../bedrock";
+
+const MAXIMUM_BRIEF_STRING_LENGTH = 17_500;
+
+const relevantResources = ["DiagnosticReport", "Patient"];
+
+function getBedrockUtilsInstance(): BedrockUtils {
+  return new BedrockUtils();
+}
+
+export async function bundleToBrief(
+  fhirBundle: Bundle,
+  cxId: string,
+  patientId: string
+): Promise<string | undefined> {
+  const { log } = out(`MR Brief for cx ${cxId}, patient ${patientId}`);
+  let briefString = await prepareBundleForBrief(fhirBundle);
+  if (!briefString) return undefined;
+
+  if (briefString.length > MAXIMUM_BRIEF_STRING_LENGTH) {
+    briefString = briefString.slice(0, MAXIMUM_BRIEF_STRING_LENGTH);
+    const msg = `Brief string input was truncated`;
+    log(msg);
+    capture.message(msg, {
+      extra: {
+        patientId,
+        cxId,
+        stringLength: briefString.length,
+        maximumLength: MAXIMUM_BRIEF_STRING_LENGTH,
+      },
+      level: "info",
+    });
+  }
+
+  const todaysDate = new Date().toISOString().split("T")[0];
+  const bedrockUtils = getBedrockUtilsInstance();
+  const prompt = `Today's date is ${todaysDate}. Write a short summary of the patient's well-being that is relevant today. Be specific with the dates for any significant events. Focus on any diagnoses that occurred in the past year.`;
```

Comment:
I think we can workshop this a tiny bit more. This is an old prompt I used to use which was good. It was for a different model but would be confident quality will be better:

`You are a senior physician. Given the attached medical documents, summarize them into a concise summary of the patients medical history. In your first sentence, include the patient's age/gender and all significant medical conditions. In the second sentence, summarize any notable recent health events (eg hospitalizations or emergency department visits) and any requested follow-up. Do not include directions on taking medications. Here is an example of an ideal summary: Patient is a 65 yo male with hx of HTN, poorly controlled diabetes, and smoking who presented with 3 hours of crushing substernal chest pain. They underwent a cardiac cath with stent placement and were discharged on aspirin and metoprolol.`

Important details:
- giving the model an expert persona. Telling LLMs they are an expert programmer, physician, etc generally improves outputs. 
- providing an example output that sounds like a clinician, using abbreviations clinicians use. It sounds like a doctors output and not a techies output.

Reply:
On the expert thing: https://arxiv.org/abs/2305.14688

Reply:
On examples (called Few Shot Learning)
https://www.ibm.com/topics/few-shot-learning#:~:text=Few%2Dshot%20learning%20is%20a,suitable%20training%20data%20is%20scarce.

Reply:
https://docs.anthropic.com/en/docs/build-with-claude/prompt-engineering/overview#how-to-prompt-engineer